src/js/xr/src/SceneManagerXR.js

```diff
@@ -133,6 +144,19 @@ const SceneContent = connect(
       scene.fog = new THREE.Fog(world.backgroundColor, -10, 40)
     }, [world.backgroundColor])
 
+    const getGpuPicker = () => {
+      if (gpuPicker.current === null) {
+        gpuPicker.current = new GPUPicker(gl)
+      }
+      return gpuPicker.current
+    }
+
+    useEffect(() => {
```

Comment:
I don't think GPUPicker init needs to move to SceneManagerXR/SceneContent. This `useEffect` could instead be moved into the interactions manager (which can get access to `scene` and `camera`).